<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XlsKeySearchTool</string>
  </property>
  <widget class="QPushButton" name="pushButton_SelectModule">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择模板文件路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectOutput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择要输出的文件路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入需要查找的列数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入要生成的序列号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_colCount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ModulePath">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_outPutPath">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_colToOutput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
